img_config.yml

```diff
@@ -1,3 +1,3 @@
 - directory: ./tiles/rooms/floor/*.*
-  width: 429.544
-  height: 256.314
+  width: 429.544px
+  height: 256.314px
```

Comment:
are you sure that adding characters to the gem config file won't err the script?

Reply:
Sorry,i didn't notice that.